Source/Protobuf/ArtifactIdDoesNotHaveDefaultParameterlessConstructor.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Dolittle.SDK.Artifacts;
+using PbArtifact = Dolittle.Artifacts.Contracts.Artifact;
+
+namespace Dolittle.SDK.Protobuf
+{
+    /// <summary>
+    /// Exception that gets thrown when an <see cref="ArtifactId" /> does not have a default parameterless constructor.
+    /// </summary>
+    public class ArtifactIdDoesNotHaveDefaultParameterlessConstructor : CouldNotConvertProtobufArtifact
```

Comment:
Same here with the `Artifact` naming but not quite sure what to call it here, `TypeId` sound very vague.